src/common/components/repository-row/index.js

```diff
@@ -33,6 +39,14 @@ const RepositoryRow = (props) => {
   );
 };
 
+const renderSnapName = (snapName) => {
+  if (!snapName) {
+    return (<a>Not registered</a>);
```

Comment:
Why `<a>` if no href?

Also - if `snapName` is null - does it mean name is not registered or only that it's not defined in snapcraft.yaml?